src/shared/components/QueryBuilder/ui/inputs/SingleEditionInput.tsx

```diff
@@ -33,72 +49,23 @@ export function SingleEditionInput({ placeholder, defaultValue, onFocus, onChang
     }
   });
 
-  // we have to implement a customized version of the input because the <Input /> from the Saga Design system does not
-  // allow any `ref` prop to be passed
   return (
-    <div className={cx(styles.wrapper, defaultValue && styles.wrapperForEdition)}>
-      <div className={styles.inputWrapper}>
-        <input
-          ref={inputRef}
-          className={styles.input}
-          placeholder={placeholder}
-          defaultValue={defaultValue}
-          onFocus={onFocus}
-          onKeyDown={onKeyDown}
-          onBlur={onBlur}
-        />
-      </div>
-    </div>
+    <Input
```

Comment:
As suggested by @ifrost in https://github.com/grafana/explore-profiles/pull/207#discussion_r1784172459